client/src/Backoffice/components/Main/components/modals/AddRide/NewRideInfo/NewRideInfo.tsx

```diff
@@ -77,6 +86,48 @@ function NewRideInfo() {
             </FormHelperText>
           )}
         </FormControl>
+
+        <FormControl>
+          <DatePicker
+            label="תאריך איסוף מבוקש"
+            defaultValue={dayjs()}
+            maxDate={dayjs().add(3, 'day')}
+            disablePast
+            onChange={(date) => {
+              setValue('pickupDateTime', date ? date.toDate() : undefined);
+            }}
+            format="YYYY-MM-DD"
```

Comment:
@aviv-taori Why are you using an unrecognizable format? I think in Israel we use this format: `DD/MM/YYYY`

Reply:
There is no use for this code, simple, this is the code you wrote, I added it. I will update and delete

Reply:
Ok

Reply:
This is your code(Only for function). But is work good!
 
![Screenshot 2024-02-25 124737](https://github.com/Ezer-Lachaim/EZLA/assets/134504658/df0678ea-71cb-4cb1-8857-7d6673345b08)